assets/scripts/gallery/view.js

```diff
@@ -196,7 +196,7 @@ export function receiveGalleryData (transmission) {
 
     // This adds the street name plaque to each thumbnail.
     // the variable is assigned, but not re-used. Do not remove!
-    var streetName = new StreetName(nameEl, galleryStreet.name)
+    var streetName = new StreetName(nameEl, galleryStreet.name) // eslint-disable-line no-unused-vars
```

Comment:
De we use `eslint`? Is there any litnter, that You use? On Travis we only use `standard` directly. Dose it use the same disabling notation that `eslint` use?